ajax/libs/datalist-polyfill/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "datalist-polyfill",
+  "description": "A minimal and dependency-free vanilla JavaScript datalist polyfill. Supports all standard's functionality as well as mimics other browsers behavior.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mfranzke/datalist-polyfill.git"
+  },
+  "author": "Maximilian Franzke",
+  "license": "MIT",
+  "filename": "datalist-polyfill.min.js",
+  "keywords": [
+    "datalist",
+    "polyfill",
+    "javascript",
+    "html",
+    "autosuggest",
+    "autosuggester",
+    "suggest",
+    "suggester",
+    "select"
+  ],
+  "npmName": "datalist-polyfill",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.js"
```

Comment:
better to use `datalist-polyfill*`
Also in old versions like in 1.0.0 there are also css files that should be hosted

Reply:
Valid point ! But similar to the above: I‘ve deprecated the CSS dependency in the early days of the project, so this version wouldn‘t be necessary any more for reference. Additionally it shouldn‘t necessarly be referenced at all, but included in their project ecosystem directly, as it‘s content is too small, so it’s „cheaper“ in relation to the savings of an extra HTTP request (even from a CDN).

Reply:
Hi @extend1994,
thanks a lot for your feedback - and totally agreed, your mentioned change makes perfectly sense to me. I've updated the PR.